nemo/collections/asr/parts/k2/graph_transducer.py

```diff
@@ -15,11 +15,17 @@
 import abc
 from contextlib import nullcontext
 from typing import ContextManager
+
 import torch
 import torch.nn.functional as F
 
 from nemo.core.classes.loss import Loss
 from nemo.core.utils.k2_guard import k2
+from nemo.core.utils.optional_libs import TRITON_AVAILABLE
```

Comment:
Can this be made into a function, similar to Numba, instead of a hardcoded constant var